services/motion/data/gps_base_with_frame.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
I personally would prefer more diversity of test input files that can be drawn upon as examples rather than One Big Config that fulfills everything; so I like the file addition. I don't feel super strongly about it though.

Reply:
If this is actually a different file then I agree, but if its just the same file with a little additional information then lets just edit the original

Reply:
this is the same file as `gps_base.json` except with frame info for the movementsensor component. I will remove this json and put frame info into the `gps_base.json` file.